src/dev/riscv/plic.cc

```diff
@@ -57,10 +57,18 @@ Plic::Plic(const Params &params) :
     PlicBase(params),
     system(params.system),
     nSrc(params.n_src),
-    nContext(params.n_contexts),
     registers(params.name, pioAddr, this),
     update([this]{updateOutput();}, name() + ".update")
 {
+    panic_if(params.hart_config != "" && params.n_contexts != 0,
```

Comment:
```suggestion
    fatal_if(params.hart_config != "" && params.n_contexts != 0,
```

This is a config issue, not a bug in gem5.